packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -1796,7 +1808,11 @@ class ThemeData with Diagnosticable {
       'https://flutter.dev/docs/release/breaking-changes/toggleable-active-color#migration-guide. '
       'This feature was deprecated after v2.13.0-0.4.pre.',
     )
-    Color? toggleableActiveColor,
+    Color? toggleableActiveColor,@Deprecated(
```

Comment:
newline needed here?

Reply:
I think one was added while I was reviewing. Please ignore this.

Reply:
😉